### PR TITLE
julia: rename build env var `MXNET_HOME` to `MXNET_ROOT`

### PR DESCRIPTION
**Pull request by iblis17**
- Add `MXNET_LIBRARY_PATH` support

Ref: https://github.com/apache/incubator-mxnet/pull/15561

### CODE REVIEW

**Comment by iblis17** [comment]
> Does this introduce some backwards incompatibility for any users?

I can make it backward compatible in this PR, and your PR will work fine.
I want to merge your PR first, and get the backward compatibility tested in this PR.

**Comment by iblis17** [comment]
okay, I added backward compatibility.

**Comment by iblis17** [comment]
okay, the build of  409ce0e looks fine.

**Comment by piyushghai** [comment]
@iblis17 Can you resolve the merge conflicts and take this PR forward ?

File: julia/deps/build.jl [hunk 1]
```diff
@@ -26,14 +26,35 @@ libmxnet_detected = false
...
 curr_win = "20190608"  # v1.5.0
 
+# TODO: remove MXNET_HOME backward compatibility in v1.7
 if haskey(ENV, "MXNET_HOME")
-  MXNET_HOME = ENV["MXNET_HOME"]
-  @info("MXNET_HOME environment detected: $MXNET_HOME")
+  @warn "The environment variable `MXNET_HOME` has been renamed, please use `MXNET_ROOT` instead."
+end
+
+# TODO: remove MXNET_HOME backward compatibility in v1.7
```
Comment (marcoabreu) [inline julia/deps/build.jl]: 2.0

Can you add it to the 2.0 tracking issue?

Comment (iblis17) [reply julia/deps/build.jl]: ah, sure (https://github.com/apache/incubator-mxnet/issues/9686#issuecomment-529131708)
```suggestion
# TODO: remove MXNET_HOME backward compatibility in v2.0
```

**Comment by aaronmarkham** [comment]
@iblis17 Hi Iblis, I'd like to revert this change since it caused the website to stop publishing. 
Let's get the new site up and apply the updates then. Also, changes will be much easier to test.

**Pull request merged**